src/main/java/duke/Storage.java

```diff
@@ -55,6 +55,7 @@ public Storage(String filePath) {
     public void saveTodo(String description) throws IOException {
```

Comment:
You could consider combining your `saveToDo(), saveEvent() and saveDeadline()` into a single method. For example:
```
public void saveToFile(String ... tokens) throws IOException {
    if (tokens.length == 1) {
        fileWriter.write(...); //todo
    } else {
        if (tokens[0].equals("event") {
            fileWriter.write(...); //event
        } else if (tokens[0].equals("deadline") {
            fileWriter.write(...); //deadline
        }
    }
}
```